mailpoet/tests/performance/utils/helpers.js

```diff
@@ -1,16 +1,20 @@
 /* eslint-disable no-shadow */
 /* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-default-export */
 /**
  * Internal dependencies
  */
 import { adminUsername, adminPassword } from '../config.js';
 /* global Promise */
 
 // WordPress login authorization
-export function login(page) {
+export function authenticate(page) {
   // Enter login credentials and login
-  page.locator('input[name="log"]').type(`${adminUsername}`);
-  page.locator('input[name="pwd"]').type(`${adminPassword}`);
+  Promise.all([
+    page.waitForNavigation({ waitUntil: 'networkidle' }),
+    page.locator('input[name="log"]').type(`${adminUsername}`),
+    page.locator('input[name="pwd"]').type(`${adminPassword}`),
+  ]);
```

Comment:
The promise here is not handled in any way. Did you mean to `await` it, for instance?

Reply:
Sorry for Git, I will follow the rule from now on.
For the Promise, should be some leftover from attempts to fix some race condition, but it works with/without Promise fine in my tests so I removed it. Thanks for the review!